README.md

```diff
@@ -67,7 +67,7 @@
 
 ## 📹 Video Description
 
-- [Video Description Link](https://www.loom.com/share/bb51379e56854720a0178d1cf8c4d2c5?sid=e109121e-20f7-49f7-b0e0-9bfd8f54962d)
+- [Video Description](https://www.loom.com/share/bb51379e56854720a0178d1cf8c4d2c5?sid=e109121e-20f7-49f7-b0e0-9bfd8f54962d)
 
 
```

Comment:
* Kindly fix the failing test i get when i run `rspec` 
![image](https://github.com/shaaibu7/Budget-app/assets/52497246/756f4d5e-04b2-405d-ad10-addc348ad7fd)

Reply:
This change has been done well